libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -1711,9 +1713,7 @@ bool AC_PolyFence_loader::write_fence(const AC_PolyFenceItem *new_items, uint16_
 
 void AC_PolyFence_loader::update() {};
 
-#if AC_POLYFENCE_FENCE_POINT_PROTOCOL_SUPPORT
```

Comment:
This is going because:-
1. we do not wrap the call to this method with this define.
2. the non-dummy methods are also not wrapped under this define